drivers/azure/azureutil/azureutil.go

```diff
@@ -481,6 +479,34 @@ func (a AzureClient) DeleteVirtualMachineIfExists(ctx context.Context, resourceG
 	return nil
 }
 
+func (a AzureClient) removeManagedDisk(ctx context.Context, resourceGroup, vmID, diskName string) error {
+	log.Debug("Attempting to remove managed disk.", logutil.Fields{"vm": vmID})
+	log.Debugf("Disk name: %q", diskName)
```

Comment:
Can we log that info in one debug statement instead of two separate?

Reply:
updated in latest commit!